editor/src/components/editor/store/dispatch.tsx

```diff
@@ -798,6 +814,141 @@ function applyProjectChangesToEditor(
   }
 }
 
+function walkJSXElementChild(
```

Comment:
this definitely does not belong in dispatch.tsx!

Reply:
Good point, and in fact we already have a `walkElement` function that can maybe be used instead https://github.com/concrete-utopia/utopia/blob/master/editor/src/core/shared/element-template.ts#L2245

Reply:
moved it and renamed it in https://github.com/concrete-utopia/utopia/pull/4786/commits/b6aa64e7e2b5e74a04dbdf89142ddd2ae676c6aa